docs/UserGuide.md

```diff
@@ -3,64 +3,54 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+ClientBook is a desktop app for managing client contacts, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, ClientBook can get your client management tasks done faster than traditional GUI apps.
+
 
 * Table of Contents
-{:toc}
 
 --------------------------------------------------------------------------------------------------------------------
+## Features
+- Viewing help : help
+- Adding a client: add
+- List all clients: list
+- Locating clients by name: find
+- Deleting a client : delete
+- Exiting the program : exit
+- Saving the data 
 
-## Quick start
```

Comment:
Do you think it would be better to give the user instructions on how to initialise/use our application? Or are there reasons why you think this should not be here?

Reply:
Added the quick start into the user guide, but release is not yet ready.